sumomongodbatlascollector/api.py

```diff
@@ -20,21 +23,21 @@ class MongoDBAPI(BaseAPI):
     def __init__(self, kvstore, config):
         super(MongoDBAPI, self).__init__(kvstore, config)
         self.api_config = self.config['MongoDBAtlas']
+        self.MAX_REQUEST_WINDOW_LENGTH = self.api_config['Collection']['MAX_REQUEST_WINDOW_LENGTH'] if self.api_config['Collection']['MAX_REQUEST_WINDOW_LENGTH'] else 3600
```

Comment:
we can replace it with single line `self.api_config['Collection'].get('MAX_REQUEST_WINDOW_LENGTH', 3600)`

Reply:
Have changes this to a .get lookup with default values.